Bmx.Core/BmxCore.cs

```diff
@@ -70,7 +62,25 @@ public class BmxCore {
 				// TODO: Remove need for input here, Push style flows have no user input on app here
 				// ex: https://developer.okta.com/docs/reference/api/authn/#response-example-waiting
 				PromptMfaInput( selectedMfa.Name );
+				throw new NotImplementedException();
 			}
+
+
+			// TODO: Decouple this more, there is an implicit understanding its an Okta AWS app
+			var accounts = await _identityProvider.GetAccounts( "amazon_aws" );
+			account = account?.ToLower();
+
+			int selectedAccountIndex = -1;
+
+			if( account != null ) {
+				selectedAccountIndex = Array.IndexOf( accounts.Select( s => s.ToLower() ).ToArray(), account );
+			}
```

Comment:
I think you can use `FirstOrDefault()` here if you want. That might be more succinct.

```cs
account = accounts
  .Select( s => s.ToLower() )
  .FirstOrDefault( s => s == account );
```

Reply:
Needed the index specifically, since within Idp.Okta the Accounts are in an array with more Okta specific information (links, Ids etc...), whereas BMX Core only needs to care about the names of the accounts (it gets a string array).

Consistent to how MFA Factors was handled.